mmv1/products/cgc/api.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: CGC
+display_name: CGC
+cgc_only: true
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: na
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: placeholder
+    url: placeholder
+objects:
+  - !ruby/object:Api::Resource
+    name: Snippet
```

Comment:
If there's a lot of snippets in the future and we wanted to separate them we could use new resources to do it, right? GKE-sized files are hard for my computer to load so I'd like to avoid adding more 😅

Reply:
Definitely a case of decide when we get there on how to split them up!

Reply:
I wanted to confirm we had the optional available from the start